okhttp/src/main/java/com/squareup/okhttp/internal/Internal.java

```diff
@@ -35,6 +38,12 @@
  */
 public abstract class Internal {
   public static final Logger logger = Logger.getLogger(OkHttpClient.class.getName());
+
+  public static void initializeInstanceForTests() {
```

Comment:
Yes. This is required for any test that interacts with anything that touches Internal without first touching OkHttpClient. Having an explicit method seemed a way of explaining that. Without it you can get to a point where it tries to use Internal.instance and finds it null.

Not typically an issue unless you are running tests individually (as I was).